packages/image_picker/image_picker_for_windows/README.md

```diff
@@ -0,0 +1,36 @@
+# image\_picker\_for\_windows
```

Comment:
We should call it `image_picker_windows`, as that's our standard naming. The "_for_web" in some packages was to work around the fact that in some cases a third party published something under the "_web" name before we added web support.